index.html

```diff
@@ -20,6 +20,9 @@ <h1>Kitten App</h1>
     <h2>Kittens in a Basket</h2>
     <img src="https://images.unsplash.com/photo-1517331156700-3c241d2b4d83?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1768&q=80" alt="a basket of orange and calico kittens in front of a lilac tree">
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin hendrerit feugiat urna, in vestibulum justo vulputate in. Curabitur blandit lobortis scelerisque. Fusce sed maximus libero, eget rutrum dui. Cras tincidunt est risus, vitae laoreet orci maximus sed. Fusce fermentum volutpat ante, congue bibendum leo pellentesque vel. Morbi nec gravida felis. Integer mattis fermentum dui, sed laoreet ipsum tempor in. Integer a turpis pellentesque, porttitor ante sit amet, feugiat augue. Mauris eget turpis nec odio commodo imperdiet. Phasellus lacinia lectus purus, at vulputate orci volutpat quis.</p>
+    <div>
```

Comment:
This div should be a footer element, it's more semantic

Reply:
thanks for catching that! just updated